doc/reference/glossary.rst

```diff
@@ -59,7 +59,7 @@ Glossary
 
       This is implemented in the Alonzo hard fork of the Cardano blockchain.
 
-      See :ref:`what_is_a_ledger`.
+      See :ref:`what_is_a_ledger`
```

Comment:
why all these changes? is it not grammatical to have a full stop at the end?

Reply:
An alternative approach would be to use a different link text in that circumstance.